### PR TITLE
Added a ToC

### PR DESCRIPTION
**Pull request by LFGaming**
Hope i've done the ToC right but also the Sign-off :)
Signed-off-by: LFGaming <38732192+LFGaming@users.noreply.github.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -19,7 +19,46 @@ At any time, community members are invited to:
 
 ## Table of Contents
 
-[ToC]
+## Table of Contents
```
Comment (moul) [inline CONTRIBUTING.md:22]: duplicate title (l20, l22)

**Review by LFGaming — COMMENTED** [review]
removed double title.

File: CONTRIBUTING.md (hunk 1 above)
Comment (LFGaming) [reply CONTRIBUTING.md]: removed the double title.

**Comment by LFGaming** [comment]
Friendly bump ;)

**Review by moul — APPROVED** [review]

**Pull request merged**